### ISSUE TITLE
Summarize repo relationships in a diagram

### ISSUE THREAD

**Issue by MichaelCurrie** [openworm/movement_cloud#94]
It turns out in February 2016 I had started working on this already:

https://www.lucidchart.com/documents/edit/544ac7aa-763a-49b8-a860-15734084f845#

@aexbrown you mentioned we should get this updated for the paper.  I will work on this.  Everyone else please feel free to make edits as needed.

**Comment by MichaelCurrie** [comment]
One issue I have with these diagrams is what relationships to depict.  Is it data flow, data relationships, or something else?

![image](https://user-images.githubusercontent.com/5267078/27207957-6ba0c344-51f7-11e7-8ce7-90d65844f68d.png)

**Comment by ver228** [comment]
Hello Michael,
I am not currently using the Tracking_Hypo.

Currently it is like:
```
experiments -> tierpsy <-> OWAT 
                  v
                 WCON
```

I guess in the future it could be something like:
```
 experiments -> (any tracker) ->  WCON -> OWAT    
```

**Comment by MichaelCurrie** [comment]
@ver288 does this mean tierpsy-tracker contains all the computer vision code to turn the videos into skeleton data?

I agree about your future state but could we provide video processing to other labs also?

` experiments -> (any tracker) -> tierpsy video processing ->  WCON -> OWAT    `

**Comment by ver228** [comment]
No, we haven't really integrated `tracking_hypo` work. I think they were some issues left.

**Comment by MichaelCurrie** [comment]
This workflow could be shared in an explanatory page on the website as a landing page for anyone interested in Worm Tracker 2.0, tierpsy tracker, WCON, or OWAT.

It would be nice to illustrate each of the one-liners needed to start these steps from the command line, as well as what to do from the website.
